datafusion/tests/sql.rs

```diff
@@ -1276,13 +1312,18 @@ async fn query_cast_timestamp_millis() -> Result<()> {
     ctx.register_table("t1", Arc::new(t1_table))?;
 
     let sql = "SELECT to_timestamp_millis(ts) FROM t1 LIMIT 3";
-    let actual = execute(&mut ctx, sql).await;
+    let actual = execute_to_batches(&mut ctx, sql).await;
+
     let expected = vec![
-        vec!["2009-03-01 00:00:00"],
-        vec!["2009-03-01 00:01:00"],
-        vec!["2009-04-01 00:00:00"],
+        "+--------------------------+",
+        "| totimestampmillis(t1.ts) |",
+        "+--------------------------+",
+        "| 2009-03-01 00:00:00      |",
+        "| 2009-03-01 00:01:00      |",
+        "| 2009-04-01 00:00:00      |",
+        "+--------------------------+",
```

Comment:
@alamb this test seems to be testing for milliseconds but the results dont have that much precision.  should the test be updated to get to the appropriate precision?  similar question for the microsecond test beneath this one.

Reply:
The idea of adding some data that actually has millisecond precision sounds like a good idea to me. I think it could also be done as another PR (perhaps file at ticket) as the tests are no worse than before